packages/monitor-v2/src/price-publisher/PublishPrices.ts

```diff
@@ -0,0 +1,150 @@
+import { paginatedEventQuery } from "@uma/common";
+import { OracleHubEthers, OracleRootTunnelEthers, VotingV2Ethers } from "@uma/contracts-node";
+import {
+  ArbitrumParentMessenger,
```

Comment:
Similar to @chrismaree's comment, can this be done like the ones on line 2?

Reply:
@mrice32 @chrismaree  I think `ArbitrumParentMessengerEthers` and `OptimismParentMessengerEthers` are not exported in  "@uma/contracts-node". We might need to do something there to add them

Reply:
Odd, fine to keep it this way then!